azure-pipelines.yml

```diff
@@ -94,3 +99,18 @@ jobs:
           testResultsFiles: '$(testResultsFiles)'
           testRunTitle: 'Windows - $(name)'
           mergeTestResults: true
+  - job: GitHub_Release
+    displayName: Release to GitHub
+    dependsOn: 
+    - Windows
+    - Linux
+    condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/v1.'))  
+    steps:
+    - task: GitHubRelease@0
+      displayName: 'GitHub release (create)'
+      inputs:
+        gitHubConnection: 'github-zenithworks'
```

Comment:
Presumably this connection would need to be replaced with something else, right?

Reply:
My bad, i mentioned this below the image - very easy to miss

> PS: For the pipeline to run correctly, it needs a service connection to be created in azure pipelines portal and the service connection needs to be passed to gitHubConnection parameter.

But you're right.  it needs a [service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) to be created in azure pipelines portal. Someone who has admin permission in the azure pipelines jupyterlab account would be able to create it. Here is how to create this:  [github service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#sep-github)